backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/attachments/AttachmentEntryDataResource.java

```diff
@@ -82,16 +81,18 @@ public boolean equals(final Object other)
 		}
 	}
 
+	@NonNull
 	@Override
 	public String getFilename()
 	{
 		return filename;
 	}
 
+	@NonNull
 	@Override
 	public String getDescription()
 	{
-		return description;
+		return CoalesceUtil.coalesceNotNull(description, "");
```

Comment:
why returning empty if null?... IMHO this shall be the responsability of the caller.